src/components/core/DpBulkEditHeader.stories.mdx

```diff
@@ -0,0 +1,50 @@
+import { Meta } from '@storybook/addon-docs'
+import DpBulkEditHeader from './DpBulkEditHeader'
+
+<Meta
+    title="Components/BulkEditHeader"
+    component={DpBulkEditHeader}
+/>
+
+# BulkEditHeader
+
+The BulkEditHeader component allows users to perform bulk editing operations on a set of data or records.
+
+In a data management system, users often need to edit multiple records simultaneously.
+Instead of editing each record individually, a bulk edit functionality allows users to select multiple records
+and apply changes to specific fields in one go.
+
+## Custom usage
+
+```html
+<div>
+  <dp-bulk-edit-header
+      v-if="selectedItemsCount.length > 0"
+      :selected-items-count="selectedItemsCount.length"
+      selection-text="Elements"
+      @reset-selection="resetSelection"
+      v-bind="args">
+    <button
+        class="btn-icns u-m-0"
+        type="button"
+        @click.prevent="deleteSelection">
+      <i class="fa fa-times u-mr-0_125" />
+      Delete
+    </button>
```

Comment:
- Can you use `DpButton` in the example?
- The `fa-times` icon should not be used for destructive actions (see the Storybook pawge for DpIcon), please use the `icon` prop of `DpButton` with the correct icon

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/615/commits/eab8db3c048044e96042470da14f70d37f88c82b

Reply:
- see comment in DpButton
- please also use DpButton here